src/console/index.tsx

```diff
@@ -5,6 +5,10 @@ import { AppProvider } from "./app/provider";
 import App from "./App";
 import "./index.css";
 
+const colorScheme = (new URLSearchParams(window.location.search)).get('colorScheme')
+/* @ts-ignore */
```

Comment:
```suggestion
/* @ts-expect-error: The colorScheme property is not on the CSSStyleDeclaration type in TypeScript v4.3.5 */
```

this way you'll get a reminder to remove it when upgrading, and it won't hide all errors

Reply:
TIL! Thank you :heart: